hub/core/dataset/dataset.py

```diff
@@ -1343,6 +1343,70 @@ def _set_read_only(self, value: bool, err: bool):
     def read_only(self, value: bool):
         self._set_read_only(value, True)
 
+    def cleanlab(
+        self,
+        module = None,
```

Comment:
I think that I'd need to use Type[Class] to specify the class that should be passed. However, I don't want to import the classes (e.g. `torch.nn.Module module`) into the `dataset.py` file to avoid the dependencies issues.